sky/tools/roll/patch.py

```diff
@@ -14,6 +14,10 @@ def patch_and_filter(dest_dir, relative_patches_dir):
       lambda line: not "gyp_environment" in line)
   utils.commit("filter gyp_environment out of build/landmines.py")
 
+  utils.filter_file("mojo/dart/embedder/BUILD.gn",
```

Comment:
I did that as part of the roll because otherwise it would cause this script to fail.